Some db connector documentation fixes (#1720)
mulesoft/docs-connectors: db/1.10/modules/ROOT/pages/database-documentation.adoc

** SECONDS
** MINUTES
** HOURS

> ** DAYS |  A time unit that qualifies the `#queryTimeout`. Values specified in nano, micro or milliseconds are rounded to seconds. | `SECONDS` |

Reviewer: Global - Values specified in nanoseconds, microseconds, or milliseconds are rounded to seconds.

> ** DAYS |  A time unit that qualifies the `#queryTimeout`. Values specified in nano, micro or milliseconds are rounded to seconds. | `SECONDS` |

Reviewer: Global in your changes - nanoseconds, microseconds, or milliseconds

** SECONDS
** MINUTES
** HOURS
** DAYS |  A time unit that qualifies the `#queryTimeout`. Values specified in nano, micro or milliseconds are rounded to seconds. | `SECONDS` |
| Fetch Size a| Number |  Indicates how many rows to fetch from the database when rows are read from a ResultSet. | 10 |
| Max Rows a| Number |  Sets the limit for the maximum number of rows that any ResultSet object generated by this message processor can contain for the given number. If the limit is exceeded, the excess rows are silently dropped. |  |
| SQL Query Text a| String |  The text of the SQL query to execute |  | x
| Parameter Types a| Array of <<ParameterType>> |  Allows you to optionally specify the type of one or more of the parameters in the query. If a value is provided, you're not required to reference all of the parameters; but you cannot reference a parameter that is not present in the input values. |  |

> | Input Parameters a| Object |  A map in which keys are the name of an input parameter to set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a colon prefix (e.g. `where id = :myParamName`).  The map's values contain the actual assignation for each parameter. |  |

Reviewer: ```suggestion
| Input Parameters a| Object |  A map in which keys are the name of an input parameter to set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a colon prefix (for example, `where id = :myParamName`).  The map's values contain the actual assignation for each parameter. |  |
```

> | Input Parameters a| Object |  A map in which keys are the name of an input parameter to set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a colon prefix (e.g. `where id = :myParamName`).  The map's values contain the actual assignation for each parameter. |  |

Reviewer: Global in your changes - e.g. > that is,

=== Select
`<db:select>`

> This operation queries data from a database. To prevent loading all the results at once, which could lead to performance and memory issues, results are automatically streamed. This means that pages of _fetchSize_ rows are loaded lazily when needed. If this operation is performed inside a transaction (e.g. within a try scope) and that transaction is closed before consuming the data, accessing the results which haven't been loaded will fail.

Reviewer: ```suggestion
This operation queries data from a database. To prevent loading all the results at once, which can lead to performance and memory issues, results are automatically streamed. This means that pages of `_fetchSize_ rows` are loaded when needed. If this operation is performed inside a transaction (that is, within a *Try Scope* component) and that transaction is closed before consuming the data, accessing the results that haven't been loaded fails.
```

> This operation queries data from a database. To prevent loading all the results at once, which could lead to performance and memory issues, results are automatically streamed. This means that pages of _fetchSize_ rows are loaded lazily when needed. If this operation is performed inside a transaction (e.g. within a try scope) and that transaction is closed before consuming the data, accessing the results which haven't been loaded will fail.

Reviewer: ```suggestion
This operation queries data from a database. To prevent loading all the results at once, which can lead to performance and memory issues, results are automatically streamed. This means that pages of `_fetchSize_ rows` are loaded when needed. If this operation is performed inside a transaction (that is, within a *Try* scope) and that transaction is closed before consuming the data, accessing the results that haven't been loaded fails.
```

`<db:stored-procedure>`

> Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, results are not read all at once. Instead results are automatically streamed to prevent performance and memory issues. This behavior means that pages of _fetchSize_ rows are loaded lazily when needed. If the *Stored procedure* operation performs inside a transaction (for example a *Try* scope component), and that transaction is closed before consuming the data, accessing the results which haven't been loaded fail.

Reviewer: ```suggestion
Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, results are not read all at once. Instead, results are automatically streamed to prevent performance and memory issues. This behavior means that pages of `_fetchSize_` rows are loaded lazily when needed. If the *Stored procedure* operation performs inside a transaction (for example, in a *Try Scope* component), and that transaction is closed before consuming the data, accessing the results that haven't been loaded fails.
```

> Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, results are not read all at once. Instead results are automatically streamed to prevent performance and memory issues. This behavior means that pages of _fetchSize_ rows are loaded lazily when needed. If the *Stored procedure* operation performs inside a transaction (for example a *Try* scope component), and that transaction is closed before consuming the data, accessing the results which haven't been loaded fail.

Reviewer: ```suggestion
Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, results are not read all at once. Instead, results are automatically streamed to prevent performance and memory issues. This behavior means that pages of `_fetchSize_` rows are loaded when needed. If the *Stored procedure* operation performs inside a transaction (for example, inside a *Try* scope component), and that transaction is closed before consuming the data, accessing the results that haven't been loaded fails.
```
